freebasics/static/css/template-base.css

```diff
@@ -0,0 +1,300 @@
+    background-color: #333;
+    max-width: 35em;
+    margin: 0 auto;
+    font-size: 10px;
+    } }
+html.template-base screen and (min-device-width: 50em){
+  html{
+    max-width: 20em; } }
+
+html.template-base body{
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  letter-spacing: 0.025em;
+  line-height: 20px;
+  font-size: 12px;
+  background-color: #efefef;
+}
+html.template-base .block-header .container-fluid{
+  margin-right: -10px;
+  margin-left: -10px;
+}
+.row:after, .row:before, .container-fluid:after, .container-fluid:before{
+  display: table;
+  content: " ";
+}
+
+p,h6{
+  margin: 0
+}
+h6{
+  font-size:13px;
+}
+.fb-card{
+  margin: 7px;
+  border-radius: 3px;
+  background-color: #fff;
+}
+.no-fb-card{
+  background-color: #fff;
+}
+.block-header{
+  color: #000;
+  background-color: #fff;
+  border-bottom: 1px solid #eeeeee;
+  overflow: hidden;
+}
+.block-logo{
+  padding: 10px;
+}
+.logo-placeholder{
+  margin: -5px 0px -5px -10px;
+  position: relative;
+  width: 30px;
+  height: 30px;
+  background-image: url(../../../static/css/img/logo_temp.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  overflow: visible;
+  float: left;
+}
+.block-logo span{
+  margin-left: 0;
+}
+.block-menu span{
+  border-left: 1px solid #eeeeee;
+  padding: 10px 0 10px 15px;
+  font-weight: 400
+}
+.block-footer{
+  color: #000;
+  background-color: #eeeeee;
+  padding: 1px 5px 5px 5px;
+}
+.block-footer p{
+  margin: 10px;
+  font-size: 11px;
+  text-align: center;
+}
+.block-header .container-fluid{
+  padding-left: 20px;
+  padding-right: 26px;
+}
+.block-article p{
+  font-size: 12px;
+  line-height: 15px;
+  padding: 0px 15px 15px 15px;
+}
+.block-default p{
+  font-size: 12px;
+  line-height: 15px;
+}
+.block-img{
+  width: 100px;
+  height: auto;
+  float: right;
+  margin: 15px;
+}
+.block-heading{
+  padding: 10px 15px;
+  font-size: 15px;
+}
+.see-more{
+  text-decoration: none;
+  text-transform: capitalize;
+  font-size: 14px;
+  font-weight: initial;
+}
+.link-button{
+  margin-top: 10px;
+  font-size: 13px;
+  text-decoration: underline;
+}
+.sub-heading{
+  padding: 15px 15px 10px 15px;
+}
+.checkbox, .radio{
+  margin-top: 0px;
+  margin-bottom: 10px;
+  font-size: 12px;
+}
+.radio label{
+  font-weight: 300;
+  line-height: 22px;
+}
+.block-poll .form-group{
+  padding: 0px 15px 15px 15px;
+  margin-bottom: 0;
+}
+.block-poll p{
+  font-size: 12px;
+  line-height: 15px;
+}
+.post-comment textarea{
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
+	width: 100%;
+}
+.block-category ul{
+  list-style: none;
+  margin:0;
+  padding:0;
+}
+.block-category li{
+  border-bottom: 1px solid #eeeeee;
+  padding: 15px 15px;
+}
+.block-category li:last-child{
+  border-bottom: none;
+}
+.block-category a{
+  text-decoration: none;
+  text-transform: capitalize;
+}
+.block-banner{
+  background-image: url(../../img/banner_palceholder_trans.png);
```

Comment:
I think these need to go.. please check that removing it doesn't cause any problems.